### DISCUSSION TITLE
How to get the "TM-30" "Rg" value?

### DISCUSSION THREAD

**Discussion by frankkjensen** [colour-science/colour#1075] [Q&A] [answered]
Hi all,

I get the TM-30 color fidelity index by: `colour.colour_fidelity_index(sd)`
How do I get the color gamut index?
It is in the TM-30 report, but how do I get it as a value?

-Frank

**Comment by thomasmansencal** [comment, marked as answer]
Hi @frankkjensen,

Please look at passing `method="ANSI/IES TM-30-18"` and then the output  specification should have `R_g`.

- https://colour.readthedocs.io/en/develop/generated/colour.colour_fidelity_index.html#colour.colour_fidelity_index
- https://colour.readthedocs.io/en/develop/generated/colour.quality.colour_fidelity_index_ANSIIESTM3018.html#colour.quality.colour_fidelity_index_ANSIIESTM3018
- https://colour.readthedocs.io/en/develop/generated/colour.quality.ColourQuality_Specification_ANSIIESTM3018.html#colour.quality.ColourQuality_Specification_ANSIIESTM3018

Cheers,

Thomas

**Reply by frankkjensen** [reply]
That's great, thanks :-)

-Frank
